tests/lib/rules/curly-bracket-spacing.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * @fileoverview Enforce spacing on the style of curly brackets.
+ * @author Armano
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/curly-bracket-spacing')
+const RuleTester = require('eslint').RuleTester
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+  parser: 'vue-eslint-parser',
+  parserOptions: { ecmaVersion: 2015 }
+})
+
+ruleTester.run('curly-bracket-spacing', rule, {
+
+  valid: [
+    {
+      filename: 'test.vue',
+      code: '<template></template>'
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div></div></template>'
+    },
+    {
+      filename: 'test.vue',
+      code: '<template>             <div id="               "></div>         </template>'
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div>{{ text }}</div></template>'
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div>{{ }}</div></template>'
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div>{{ }}</div></template>',
+      options: ['always']
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div>{{}}</div></template>',
+      options: ['never']
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div>{{text}}</div></template>',
+      options: ['never']
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div>{{ text }}</div></template>',
+      options: ['always']
+    }
+  ],
+
+  invalid: [
+    {
+      filename: 'test.vue',
+      code: '<template><div>{{         }}</div></template>',
```

Comment:
I think that this rule should ignore mustaches which have syntax errors. If syntax errors exist, `node.expression` is `null`.